.travis.yml

```diff
@@ -101,12 +101,16 @@ script:
   - |
     if [[ -v HIVE_TESTS ]]; then
       presto-hive-hadoop2/bin/run_hive_tests.sh
-      if [[ -v S3_TESTS_AWS_ACCESS_KEY_ID ]]; then
-        env AWS_ACCESS_KEY_ID=$S3_TESTS_AWS_ACCESS_KEY_ID \
-        AWS_SECRET_ACCESS_KEY=$S3_TESTS_AWS_SECRET_ACCESS_KEY \
+      if [[ -v HIVE_TESTS_AWS_ACCESS_KEY_ID ]]; then
```

Comment:
Not sure where these are set in Travis but the same access keys can be reused for Glue.